x-pack/solutions/observability/packages/utils_server/entities/get_data_streams_for_entity.ts

```diff
@@ -54,7 +54,16 @@ export async function getDataStreamsForEntity({
   });
 
   const dataStreams = uniq(
-    compact(await resolveIndexResponse.indices.flatMap((idx) => idx.data_stream))
+    compact([
+      ...resolveIndexResponse.indices.flatMap((idx) => {
```

Comment:
can you leave a comment here explaining why we check both `indices` and `data_streams`?